mozilla/bedrock
# Initial support for Pocket l10n using Fluent strings in Bedrock

## One-line summary

This changeset lays in a skeleton l10n directory and subdirs, giving us somewhere to put Fluent files as we mark up Pocket templates for l10n.

It also marks up the `pocket/about.html` page (and the related nav, mobile nav and footer template partials) with Fluent strings. 

This marking up of the `about` page also shows we can pull in brand names from the default mozorg l10n files. 

In general, strings in pocket pages are namespaced with `pocket-` and don't clash with existing strings

## Issue / Bugzilla link

#11519

# Testing

* Pull this branch
* Start the site in Pocket Mode - either `npm run in-pocket-mode` or `make run-pocket` 
* Go to `/about/` and confirm that
- [x] the page shows only English strings, with no Fluent placeholders visible.
- [x] all hyperlinks work as you'd expect
- [x] the mobile and desktop navs work as you'd expect
(You can compare with https://getpocket.com/en/about/)
